virttest/bootstrap.py

```diff
@@ -955,7 +956,6 @@ def bootstrap(options, interactive=False):
                 LOG.debug("Module %s loaded", module)
 
     LOG.info("")
-    LOG.info("VT-BOOTSTRAP FINISHED")
-    LOG.debug("You may take a look at the following online docs for more info:")
-    LOG.debug(" - http://avocado-vt.readthedocs.org/")
-    LOG.debug(" - http://avocado-framework.readthedocs.org/")
+    LOG.info("vt-bootstrap finished")
```

Comment:
It's really a change of your preference against mine.  My reason is that the last message is special, and should be obvious and easy to grasp after some ~ 100 lines of bootstrap related messages (wild guess).

I also consider documentation about Avocado itself to be (at the very least) as useful as documentation on Avocado-VT for users finishing a bootstrap.

So, while there's nothing wrong with your patch, I just won't ACK this because your points did not change my original understanding of the reasons that made me propose those changes in the first place.